posthog/hogql_queries/web_analytics/web_overview.py

```diff
@@ -320,6 +336,13 @@ def previous_period_aggregate(function_name, column_name, alias):
                 current_period_aggregate("avg", "is_bounce", "bounce_rate"),
                 previous_period_aggregate("avg", "is_bounce", "prev_bounce_rate"),
             ]
+            if self.query.includeLCPScore:
+                select.extend(
+                    [
+                        current_period_aggregate("quantiles", "lcp", "lcp_p90", params=[ast.Constant(value=0.9)]),
```

Comment:
@Lior539 recommended we do p75 of web vitals apparently that matches other tools

Reply:
Ah nice, I can see the same recommendation on web.dev https://web.dev/articles/lcp . Not sure where I got p90!